src/tasks/LargeFileUploadTask.ts

```diff
@@ -50,6 +54,7 @@ export interface LargeFileUploadTaskOptions {
 export interface LargeFileUploadSession {
 	url: string;
 	expiry: Date;
+	isCancelled?: boolean;
```

Comment:
why is a nullable needed here? simply use a boolean (non-nullable) which will default to false

Reply:
> No. You cannot provide default values for interfaces or type aliases as they are compile time only and default values need runtime support
https://stackoverflow.com/questions/35074365/typescript-interface-default-values

Can you please give me an example if you are suggesting some else?

Reply:
As far as I understand, you don't need to set that default value. The default of bool (non-nullable) is false. Just the only change here would be to remove the ? sign.

Reply:
The point of making this parameter optional is that in most cases the `isCancelled status`  will be false. Removing the ? makes it necessary for the user to specify for the user everytime a `LargeFileUploadSession` is created. The `LargeFileUploadSession` is created and passed when created a large file upload task and the status `isCancelled` will always be false in this case.

Reply:
right. Sorry I forgot that in some cases typescript would enforce providing non-nullable properties on objects. I believe one could get around using deconstruct and/or assign but that gets messy.
Thanks for the clarification and sorry about the mumbling here 😂